README.md

```
@@ -1,2 +1,25 @@
 # sleap-roots-training
 Jupyter notebooks for training and evaluating models for use in the sleap-roots-pipeline. Weights and Biases logging is integrated.
+
+See (guide)[https://www.notion.so/SLEAP-Roots-Model-Training-Guide-21b4a67a766780ec91c0f92ae459ef2c?source=copy_link]
```

markdownlint-cli2 (0.17.2)

Reversed link syntax
(guide)